src/resources/service.test.ts

```diff
@@ -223,3 +233,46 @@ it('can conditionally not set a filter', async () => {
 
   expected.toHaveProperty('filters', {});
 });
+
+it('will get services for specified location when both location and region filters are provided', async () => {
+  const resource = new Service(mockAxios);
+
+  await resource
+    .at(1)
+    .located({ region: 'SK' })
+    .sortBy('created')
+    .take(5)
+    .on(1)
+    .get();
+
+  expect(mockAxios.get).toHaveBeenCalledTimes(1);
+  expect(mockAxios.get).toHaveBeenCalledWith('services', {
+    params: {
+      'filter[location]': 1,
+      limit: 5,
+      page: 1,
+      sort: 'created',
+    },
+  });
+});
```

Comment:
What is the reasoning behind not supplying both parameters if they are both supplied? I don't think this package should be laying down rules like this.

Reply:
I have removed the unnecessary logic and the associated tests